simply/actor.py

```diff
@@ -626,26 +628,27 @@ def generate_order(self):
         self.orders.append(new)
         return new
 
-    def get_price(self, index, final_price, energy):
+    def get_price(self, steps, final_price, energy):
         """ Return the price for order generation for a planned future order generation.
 
-        Planned order generation can be moved forward if energy storage is possible. Since urgency
-        for energy procurement is lower prices can be adjusted in a favorable way for the actor.
-        This method returns the price based on the actor.attribute pricing strategy and takes
-        the input of index which is the number of time steps until an order will be placed,
-        the final_price which would be used for this order and the energy of this order.
-        Positive energy values stand for buying of energy, i.e. a reduced price will be generated.
+        Planned order generation can be moved to an earlier time if energy storage is possible.
+        Since urgency for energy procurement is lower, prices can be adjusted favorably for the
+        actor. This method returns the current price based on the actor's attribute
+        :py:attr:`~simply.actor.pricing_strategy` and a planned future order. Parameters are
+        number of time steps until the future order will be placed, the final_price which would be
+        used for this future order and the energy of this future order.
+        Positive energy values stand for buying of energy, i.e. a lower price will be generated.
```

Comment:
actually parameter description is redundant with list below, i.e. can be deleted. Some added formulation can be moved below i.e. "will be placed" and ", i.e. a lower price will be generated"